docs/usage-as-end-user.md

```diff
@@ -16,12 +16,53 @@ Applications can use AccessKey pair to authorize themselves to a set of APIs and
 
 Make sure to [create a Resource Access Management User](https://www.alibabacloud.com/help/doc-detail/93720.htm), and [create an AccessKey Pair](https://partners-intl.aliyun.com/help/doc-detail/116401.htm) that shall be used for the Shoot cluster.
 
-[Grant at least the following permissions](https://partners-intl.aliyun.com/help/doc-detail/116146.htm) to the Resource Access Management User.
-- AliyunECSFullAccess
-- AliyunVPCFullAccess
-- AliyunSLBFullAccess
-- AliyunNATGatewayFullAccess
-- AliyunEIPFullAccess
+### Permissions
+
+Please make sure the provided credentials have the correct privileges. You can use the following Alicloud RAM policy document and attach it to the RAM user backed by the credentials you provided.
+```
+{
```

Comment:
@minchaow , can you update also https://github.com/gardener/dashboard/blob/25ff084247b7e640e9b22037b0167bc2bd4084ad/frontend/src/components/dialogs/SecretDialogAlicloudHelp.vue#L58-L78?